src/styles.scss

```diff
@@ -1,17 +1,18 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
 @import "./styles/variables.scss";
 @import "./components/Navbar/styles.scss";
+@import "./components/CharacterList/styles.scss";
```

Comment:
Import component styles inside components' own file, no need to import all of them here.